.github/workflows/test-osbuild-composer-integration.yml

```diff
@@ -0,0 +1,161 @@
+---
+name: "osbuild-composer integration"
+
+# PLEASE UPDATE THIS COMMENT IF ANY RELEVANT CHANGES ARE MADE TO THE WORKFLOW
+#
+# This workflow tests if an open PR breaks osbuild-composer's compatibility
+# with images. If it does, it posts a message on the PR itself notifying the
+# author and reviewers of this change. The workflow works as follows:
+#
+# 1. Checks out osbuild/osbuild-composer
+# 2. Replaces the osbuild/images dependency with the *merge base* of the open
+#    PR with main and runs osbuild-composer's unit tests.
+# 3. If the unit tests on the merge base (step 2) succeed, replaces the
+#    osbuild/images dependency with the *HEAD* of the open PR and runs
+#    osbuild-composer's unit tests. If the tests on the merge base failed, no
+#    further action is taken.
+# 4. At most one of two messages is posted:
+# 4.1 Posts a message on the open PR only if the unit tests with the merge base
+#     (step 2) succeed and the unit tests with the PR HEAD (step 3) fail. This
+#     combination of outcomes indicates that the PR is the one responsible for
+#     the breakage.
+# 4.2 Updates the existing message on the open PR only if the unit tests with
+#     the merge base (step2) succeed, the unit tests with the PR HEAD (step 3)
+#     succeed, and there is already a message posted by this workflow. This is
+#     meant for cases where a PR initially breaks compatibility and then it gets
+#     fixed. No message should be posted on a PR that doesn't affect the
+#     integration tests.
+#
+# Limitations:
+# 1. This workflow runs on pull_request_target, which means it runs on the main
+#    branch. Changes to this workflow in a PR will not affect the run for that
+#    PR. Running on pull_request_target is needed to have access to repository
+#    secrets (Schutzbot's GitHub token).
+# 2. If the unit tests in this repository fail, the integration will fail and
+#    the message will be posted (if the integration is not failing on the merge
+#    base). This will happen even if there's no actual integration issue, so
+#    the message can be misleading. However, subsequent runs that fix the issue
+#    will update the message accordingly.
+
+
+on:  # yamllint disable-line rule:truthy
+  pull_request_target:
+    branches:
+      # skip test for backport branches since it doesn't make sense to test
+      # those against osbuild-composer main
+      - main
+
+jobs:
+
+  unit-tests:
+    name: "🛃 Unit tests"
+    runs-on: ubuntu-20.04
```

Comment:
I think we could as well use ubuntu 24.04 / ¿latest? or is there a reason for taking the pre-pre LTS?

Reply:
No reason, just copy-pasta.

Reply:
Thanks.  Changed to latest like we use everywhere else.